SECURITY.md

```
@@ -0,0 +1,94 @@
+receiving such patches depends on the CVSS v3.0 Rating:
+
+| CVSS v3.0 | Supported Versions                        |
+| --------- | ---------------------------------------- |
+| 9.0-10.0  | Releases within the last 6 months        |
+| 4.0-8.9   | Most recent release                      |
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities through private means:
+
+1. Email: security@securepass.generator
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)